BlogAPI.Persistence/Models/Comment.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace BlogAPI.Persistence.Entities
+{
+    public class Comment
+    {
+        public int Id { get; set; }
+        public string Author { get; set; }
+        public string Content { get; set; }
+        public DateTime CreationDate { get; set; }
+        public BlogPost BlogPost { get; set; } = new BlogPost();
```

Comment:
Why does a comment contain a `BlogPost` object? This could cause `Comment` object to get very big with many comments. I think `BlogPostId` should be enough to get the original blog post.

Reply:
It's just extra navigation in case I need it later, EF Core provides an option not to include related entities, so shouldn't be an issue